app/models/user.rb

```diff
@@ -114,8 +114,11 @@ def exempt_from_2sv?
     reason_for_2sv_exemption.present?
   end
 
-  def event_logs
-    EventLog.where(uid:).order(created_at: :desc).includes(:user_agent)
+  def event_logs(event_type = nil)
+    conditions = { uid: }
+    conditions[:event_id] = event_type.id unless event_type.nil?
+
+    EventLog.where(conditions).order(created_at: :desc).includes(:user_agent)
```

Comment:
Minor: While I can that the name of `EventLog` and its `event_id` attribute are confusing (and should probably be changed!), I wonder whether it might add to the confusion to use `event_type` here rather than `event_id`...?